app/views/shared/check_answers/_income_result.html.erb

```diff
@@ -1,47 +1,30 @@
 <section class="print-no-break">
-  <% read_only = false unless local_assigns.key?(:read_only) %>
-
-  <dl id="income-result-questions" class="govuk-summary-list govuk-!-margin-bottom-9">
-    <%= check_answer_link(
-          name: :total_gross_income_assessed,
-          question: t(".total_gross_income"),
-          answer: gds_number_to_currency(@legal_aid_application.cfe_result.combined_total_gross_income_assessed),
-          read_only:,
-        ) %>
-
-    <%= check_answer_link(
-          name: :total_disposable_income_assessed,
-          question: t(".total_disposable_income"),
-          answer: gds_number_to_currency(@legal_aid_application.cfe_result.total_disposable_income_assessed(partner: @legal_aid_application&.applicant&.has_partner_with_no_contrary_interest?)),
-          read_only:,
-        ) %>
-
-    <%= check_answer_link(
-          name: :gross_income_limit,
-          question: t(".gross_income_limit"),
-          answer: number_to_currency_or_na(@legal_aid_application.cfe_result.gross_income_upper_threshold),
-          read_only:,
-        ) %>
-
-    <%= check_answer_link(
-          name: :disposable_income_lower_limit,
-          question: t(".disposable_income_lower_limit"),
-          answer: gds_number_to_currency(@legal_aid_application.cfe_result.disposable_income_lower_threshold),
-          read_only:,
-        ) %>
-
-    <%= check_answer_link(
-          name: :disposable_income_upper_limit_limit,
-          question: t(".disposable_income_upper_limit"),
-          answer: number_to_currency_or_na(@legal_aid_application.cfe_result.disposable_income_upper_threshold),
-          read_only:,
-        ) %>
-
-    <%= check_answer_link(
-          name: :disposable_income_contribution,
-          question: t(".disposable_income_contribution"),
-          answer: gds_number_to_currency(@legal_aid_application.cfe_result.income_contribution),
-          read_only:,
-        ) %>
-  </dl>
+  <%= govuk_summary_list(card: { title: heading },
+                         actions: false,
```

Comment:
If this is check_answers, should they not have change links?
I'd expect the reports to not have change links.
The deleted code had ```<% read_only = false unless local_assigns.key?(:read_only) %>``` to check

Reply:
@colinbruce Looks like the only place this is ever called is in partials in the means report, which is why change links are gone. With that in mind, do you think this partial would be better off in the `/means_reports` folder instead?